pkg/inspectors/subjectispodforuser/subjectispodforuser_test.go

```diff
@@ -0,0 +1,248 @@
+package subjectispodforuser_test
+
+import (
+	"github.com/proofpoint/kapprover/pkg/inspectors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	certificates "k8s.io/api/certificates/v1beta1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	_ "github.com/proofpoint/kapprover/pkg/inspectors/group"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func TestInspect(t *testing.T) {
+	key, err := rsa.GenerateKey(rand.Reader, 1024)
+	require.NoError(t, err, "Generate the private key")
+
+	for _, testcase := range []struct {
+		name            string
+		inspectorConfig string
+		expectMessage   string
+		serviceAccount  string
+		setupRequest    func(request *x509.CertificateRequest)
+		podNamespace    string
+		podIp           string
+	}{
+		{
+			name:          "CnHasO",
+			expectMessage: "Subject has more than one name component",
```

Comment:
Shoudn't this be "subject does not have any name component"?

Reply:
No, it has the default CommonName of "172-1-0-3.somenamespace.pod.cluster.local" plus the Organization added by the setupRequest